<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInfraProp</class>
 <widget class="QDialog" name="CInfraProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infrastructure diagram properties</string>
  </property>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>81</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width (metters)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>111</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (metters)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnWidth">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>81</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>100</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnHeight">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>111</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>100</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>151</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bacground Color</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbBack">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>151</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Gray</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSelectColor">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diagram name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
